src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -102,6 +108,19 @@ const DashboardAuthContainer = props => {
   )
 }
 
+const RightBoxSection = ({ title, initialOpenState, onClick, children }) => {
+  return div({ style: { paddingTop: '1rem' } }, [
+    div({ style: Style.dashboard.rightBoxContainer }, [
+      h(Collapse, {
+        title: h2({ style: Style.dashboard.collapsibleHeader }, [title]),
```

Comment:
I think this should be h3 because the workspace path in the titlebar area is already an h2.

Reply:
Resolved 6a5f1d8a0a821002d4eb96b5196628912a2e0c25